toolchain/lowering/lowering_context.cpp

```diff
@@ -72,13 +71,30 @@ auto LoweringContext::BuildFunctionDeclaration(SemIR::FunctionId function_id)
   // TODO: Lower type information for the arguments prior to building args.
   auto param_refs = semantics_ir().GetNodeBlock(function.param_refs_id);
   llvm::SmallVector<llvm::Type*> args;
-  args.resize_for_overwrite(first_param + param_refs.size());
+  llvm::SmallVector<SemIR::NodeId> param_nodes;
```

Comment:
Is `args` a mistaken choice of names here? Maybe `params` or `param_types` would've been better? It's particularly the `args` versus `param_nodes` that makes it feel like a mismatch to me, though I'm sure `llvm::Function` using `Arguments` influenced the original naming...

Reply:
Yeah, I'm not sure there's an ideal solution here, but renaming to `param_types` seems better. Done.

It's unfortunate that there are two different prevailing sets of terminology here -- "parameter" versus "argument", and "formal argument" versus "actual argument" -- and that LLVM picks the latter convention and shortens the terms to `Argument` and `Arg` respectively (!!), whereas language specifications (including Carbon's) usually use the former convention for clarity.